src/site/layout/mod.rs

```diff
@@ -0,0 +1,76 @@
+mod footer;
+mod head;
+mod header;
+
+use crate::config::analytics::{self, Analytics};
+use crate::config::theme;
+use crate::config::Config;
+use crate::errors::*;
+use crate::site::artifacts;
+
+use axohtml::elements;
+use axohtml::{html, text};
+
+pub struct Layout {
+    head: head::Head,
+    theme: String,
+    analytics: Option<Box<elements::script<String>>>,
+    google_script: Option<Box<elements::script<String>>>,
```

Comment:
Do you perfer importing it like this? I find it makes things more confusing

Reply:
there are name conflicts between modules and the types exported by axohtml

Reply:
Fair